testsuite/test_thumbnail.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from snoop.data.analyzers import thumbnails
+from conftest import TESTDATA, CollectionApiClient
+
+pytestmark = [pytest.mark.django_db]
+
+
+def test_thumbnail_job(fakedata, taskmanager, client):
+    test_doc = TESTDATA / './no-extension/file_doc'
+    with test_doc.open('rb') as f:
+        thumbnails.call_thumbnails_service(f, 100)
```

Comment:
Split this test into two:

- `test_thumbnail_service` that is this function
- `test_thumbanil_task` that does something else:
  - create a Blob object like here https://github.com/liquidinvestigations/hoover-snoop2/blob/229d8327b5508d7fe8e0ab82717dc6f2eaa7fbf2/testsuite/test_blobs.py#L9-L10
  - runs the Task function (the one annotated with @snoop_task) instead of the `call_thumbnails` service
  - checks that all your Thumbnail rows are there: `assert models.Thumbnail.objects.get(size=100, blob=X).thumbnail.size > 0`

None of these 2 tests requires `taskmanager` or `client` or `fakedata` since those are meant to be used directly


Here's another